src/gtc/oir.py

```diff
@@ -112,8 +112,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
 
 
 class FieldDecl(Decl):
-    # TODO dimensions
-    pass
+    dimensions: Tuple[bool, bool, bool] = (True, True, True)
```

Comment:
```suggestion
    dimensions: Tuple[bool, bool, bool]
```
This is inconsistent with GTIR

Reply:
I think this is leftover from earlier and not actually used anywhere. will change